zodable-gradle-plugin/src/main/kotlin/digital/guimauve/zodable/ZodablePlugin.kt

```diff
@@ -152,6 +153,8 @@ abstract class ZodablePlugin : Plugin<Project> {
                             listOf(
                                 "npx", "tsc", "--init",
                                 "-d",
+                                "--module", "ES2020",
+                                "--moduleResolution", "node",
```

Comment:
Typescript docs indicate this should not be used for new projects (https://www.typescriptlang.org/docs/handbook/modules/reference.html#node10-formerly-known-as-node).

I would imagine `bundler` might be more appropriate for code that is going to target RN or the web?

Also note this from the docs:

> --moduleResolution bundler must be paired with --module esnext or --module preserve.